juju/python-libjuju
# Bump allwatcher versions for 29

#### Description

Small change to add  the  `AllWatcher` facade versions we support in juju `2.9`. Schema seems to be up to date with that, so no need to recreate clients etc.

#### QA Steps

The tests need to pass. Additionally the following should work fine:

```python
from juju.model import Model
import asyncio

async def juju_stats():
    model = Model()
    await model.connect()
    print(model.is_connected())
    status = await model.get_status()
    print(status)
    await model.disconnect()

asyncio.run(juju_stats())
```

- [ ] Before landing: check and confirm that we actually support the newly added features in `AllWatcher` versions 2 and 3.
